templates/start_calcOW_template.jl

```diff
@@ -136,7 +136,8 @@ end
     pm_array = nothing # Array can be specified manually. Otherwise, leave as 'nothing'
     pm_min = -1.0
     pm_max = 1.0
-    reaction = "D(d,n)3He" # Specified on the form a(b,c)d where a is thermal ion, b is fast ion, c is emitted particle and d is the product nucleus. However, if analyticalOWs==true then 'reaction' should be provided in the format 'proj-X' where 'X' is the fast ion species ('D', 'T' etc)
+    product_state = "1L" #!# can be nothing, GS, 1L, 2L. Nothing means Ground State

```

Comment:
Remove this "Nothing means Ground State" comment? It only confuses the user? I would suggest the line to be
`product_state = "GS" # Can be GS, 1L or 2L. GS means ground state (default), 1L means first excitation level and second excitation level`